catalog/istio-stack/config/destinationrules.yaml

```diff
@@ -0,0 +1,78 @@
+---
+################################################################################
+# Istio mTLS between Apps
+################################################################################
+apiVersion: networking.istio.io/v1beta1
+kind: DestinationRule
+metadata:
+  name: istio-apps-mtls
+  namespace: apps
+spec:
+  host: "*.apps.svc.cluster.local"
```

Comment:
what is apps? What if the customer doesn't have an apps namespace? Imo this is user specific configuration whcih you can't foresee

Reply:
maybe it can be removed and a howto on the readme can be added for enabling this feature?

Reply:
i think we shouldn't do it all. This too opinionated. We just provide preconfiguration for istio based on where its running, but now how they should define their ingress/egress